main/streams/memory.c

```diff
@@ -370,6 +370,10 @@ static size_t php_stream_temp_write(php_stream *stream, const char *buf, size_t
 
 		if (memsize + count >= ts->smax) {
 			php_stream *file = php_stream_fopen_tmpfile();
+			if (file == NULL) {
+				php_error_docref(NULL TSRMLS_CC, E_WARNING, "Unable to create temporary file.  Check permissions in temporary files directory.");
+				return NULL;
```

Comment:
@smalyshev ok i will fix on the master tree. since @laruence  said fix will not be applied to 5.4